onnx/common/ir_pb_converter.cc

```diff
@@ -498,6 +538,18 @@ void addAttribute(ONNX_NAMESPACE::NodeProto* n_p, Node* n, Symbol name) {
         encodeGraph(g, v);
       }
       break;
+    case AttributeKind::tp: {
+      attr->set_type(ONNX_NAMESPACE::AttributeProto_AttributeType_TYPE_PROTO);
+      auto tp = attr->mutable_tp();
+      encodeTypeProto(tp, n->tp(name));
+    } break;
+    case AttributeKind::tps:
+      attr->set_type(ONNX_NAMESPACE::AttributeProto_AttributeType_TYPE_PROTOS);
+      for (auto& v : n->tps(name)) {
+        auto tp = attr->add_type_protos();
+        encodeTypeProto(tp, v);
+      }
+      break;
```

Comment:
nit: L490-L552 Could we sync all the bracket and break with the same coding style? It looks a little confusing.

Reply:
Run clangformat?

Reply:
@neginraoof if using Visual Studio, configure it to automatically pickup .clangformat file in this repo and other related options.